cla_public/translations/en/LC_MESSAGES/messages.po

```diff
@@ -1550,6 +1550,10 @@ msgstr ""
 msgid "We’ll give you the CLA phone number when you submit your details below."
 msgstr ""
 
+#: cla_public/templates/contact.html:114
+msgid "You can enter up to %(cookie_settings_url)s characters"
```

Comment:
Judging solely based on the variable name, is `cookie_settings_url` correctly referencing what we want here?

Reply:
As above.  Fixed.